radar/platforms-and-services/azure-api-management.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Azure API Management Service"
+ring: adopt
+quadrant: "platforms-and-aoe-services"
+tags: [cloud, azure, compute]
```

Comment:
why `compute` ?

Reply:
I've chose `compute` for [its internal computer platform](https://learn.microsoft.com/en-us/azure/api-management/compute-infrastructure). `networking` seems reductive to me